src/docs/cookbook/design/fonts.md

```diff
@@ -132,10 +122,10 @@ Text(
 
 ### TextStyle
 
-If a [`TextStyle`][] object specifies a weight
-or style for which there is no exact font file,
-the engine uses one of the more generic files for the font and attempts to
-extrapolate outlines for the requested weight and style.
+Nếu đối tượng [`TextStyle`][] có độ dày, các kiểu chữ
+không đúng với fonts đã khai báo,
+máy sẽ dùng một trong những font chữ mặc định và cố gắng
```

Comment:
Keep "engine"? @lightkeima @truongsinh

Reply:
It refers to the flutter engine I guess. Then keep it

Reply:
vậy nó sẽ là "engine sử dụng các file thông thường và có phản hồi lại các yêu cầu về độ dày và kiểu chữ sau nhiều lần truy cập fonts" ??